### PR TITLE
Disable outline on focussed dropdown-trigger

### PR DESCRIPTION
**Pull request by JayDijkstra**
## Proposed changes

As stated in https://github.com/materializecss/materialize/issues/72 the browser puts an ugly outline for an dropdown-trigger which recieves the :focus state. To fix this disable the outline for focussed .dropdown-trigger elements.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist:

- [x] I have read the **[CONTRIBUTING document](https://github.com/materializecss/materialize/blob/master/CONTRIBUTING.md)**.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by wuda-io — APPROVED** [review]
Looks good to me 👍

**Comment by Smankusors** [comment]
hmm wait wait, for accessbility reasons, is this will be a temporary solution? for someone who only use keyboard, they will find this difficult

**Comment by DanielRuf** [comment]
> hmm wait wait, for accessbility reasons, is this will be a temporary solution? for someone who only use keyboard, they will find this difficult

Exactly. We should not remove but style this.

File: sass/components/_dropdown.scss [hunk 1]
```diff
@@ -81,5 +81,9 @@ body.keyboard-focused {
 }
 
 .dropdown-trigger {
+  &:focus {
+    outline: 0;
```
Comment (DanielRuf) [inline sass/components/_dropdown.scss:85]: Please see https://www.google.com/search?channel=fs&client=ubuntu&q=outline+0+bad why this is a bad practice to remove it in general.

**Review by Smankusors — CHANGES_REQUESTED** [review]
sorry, but the PR should be about fixing the outline style, not disabling it entirely...

---

Edit : I'm just realised the original issue is not about the styling. But more about the style being applied without using keyboard.

**Comment by wuda-io** [comment]
I understand that navigating with keyboard only is difficult. But as far as i know, does not even a single button has a focus style, so i thought this would be no big deal.
I think we should disable the outline style, but change the focus to a slightly different color style. But this would be another issue i guess.
Best regards

**Comment by wuda-io** [comment]
Yea I just thought again about your concern and I must admit that you are right... :P
I looked into the main page of materializecss and there the Focus Style works.
What should we do now?
Greets

**Comment by Smankusors** [comment]
I'm sorry, but I will close this PR now.

If you decided to fix the bug mentioned in #72, feel free to create a new PR about it. 😉

**Pull request closed without merging**